### ISSUE TITLE
Configuration details better on Manage display forms?

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-contrib/title_display#6]
It's great to see this module which adds useful title display options to Backdrop's content types. I understand that the general availability of title display options has to be enabled on the content type configuration form. I'm curious why the option details are also configured there. I would rather expect them on the single *Manage display* forms, e.g. on `admin/structure/types/manage/post/display/teaser`. Are there technical limitations which prevent this?

**Comment by quicksketch** [comment]
Yep, technical limitations. Only fields can have settings. The core issue is https://github.com/backdrop/backdrop-issues/issues/3859

**Comment by olafgrabienski** [comment]
Thanks! So, I'm tending to leave this issue open as a reminder to adapt the module when the core issue is solved. Feel free to close if you don't agree!
